src/lib.rs

```diff
@@ -52,4 +52,7 @@ pub mod stream;
 pub mod sync;
 pub mod task;
 
+#[cfg_attr(feature = "docs", doc(cfg(unstable)))]
```

Comment:
This probably appears properly in the documentation, but it seems that you can actually access this module without the unstable feature. (it needs to add `#[cfg(feature = "unstable")]`)

Reply:
Done!